policy.md

```diff
@@ -219,7 +230,7 @@ Reports uploaded to the CCADB MUST be publicly available and contain at least th
 11. Date the audit statement was issued (referred to as the "issuing date" by the ACAB'c Audit Attestation Letter templates and "report date" by the WebTrust Illustrative Reports);
     - For revised or corrected versions of audit statements made to address errors, omissions, or formatting to a previously completed audit statement, the CA Owner may enter the date of the initial issuance, provided that the scope and audit period covered by the revised audit statement remain unchanged. In such cases, the audit statement should contain the initial issuance date and the subsequent issuance date and the CA Owner should provide the URL of the most recent version and include a comment in the case explaining the revision date and the reason for reissuance of the audit statement.
 12. For ETSI, a statement to indicate if the audit was a full audit, and which parts of the criteria were applied, e.g. DVCP, OVCP, NCP, NCP+, LCP, EVCP, EVCP+, QCP-w, Part1 (General Requirements), and/or Part 2 (Requirements for trust service providers), and a statement to indicate that the auditor referenced the applicable CA/Browser Forum criteria and the version used;
-13. All incidents disclosed by the CA Owner, or reported by a third party, and all findings reported by an auditor, that, at any time during the audit period, occurred, were open in Bugzilla, or were reported to a Root Store Operator; and
+13. All [incidents](#61-disclosing-and-responding-to-incidents) disclosed by the CA Owner, or reported by a third party, and all findings reported by an auditor, that, at any time during the audit period, occurred, were open in Bugzilla, or were reported to a Root Store Operator. For each, auditors SHOULD opine that (a) the scope, impact, and root cause of incidents are accurately and fairly stated in the publicly-disclosed incident reports, and (b) that the corresponding actions taken by the CA Owner satisfactorily address those root causes and meaningfully reduce likelihood of the issue’s recurrence; and
```

Comment:
I'd rephrase this to sound more neutral by replacing the word "opine" and also less predictive in terms of remediation ("meaningfully reduce likelihood of recurrence"). Here is some alternative language we could consider.
"For each, auditors SHOULD review the publicly-disclosed incident reports for consistency with audit evidence obtained" ... "and indicate [whether/that] (a) the scope, impact, and root cause are accurately and fairly described;" and "[whether/that] (b) the corrective actions described by the CA Owner are aligned with the factors that led to the incident and are intended to mitigate the risks associated with the identified root cause(s)."